src/runtime_src/core/common/info_platform.cpp

```diff
@@ -151,7 +151,7 @@ add_p2p_info(const xrt_core::device* device, ptree_type& pt)
 void
 add_performance_info(const xrt_core::device* device, ptree_type& pt)
 {
-  const auto mode = xrt_core::device_query_default<xq::performance_mode>(device, 3);
+  const auto mode = xrt_core::device_query_default<xq::performance_mode>(device, 0);
```

Comment:
0 as well as 3 are magic.  Do you maybe mean `static_cast<int>(performance_mode::power_type::basic)` ?

Reply:
yes, it was supposed to be the enum